webapp/src/modules/mortgage/sagas.js

```diff
@@ -230,5 +232,8 @@ function* handleClaimMortgageResolutionRequest(action) {
 }
 
 function* handleFetchParcelSuccess(action) {
-  yield put(fetchActiveParcelMortgagesRequest(action.x, action.y))
+  const loadingState = yield select(state => getLoading(state))
+  if (!isLoadingType(loadingState, FETCH_ACTIVE_PARCEL_MORTGAGES_REQUEST)) {
```

Comment:
Can we use the `take` instead of `takeLatest` effect to replace this? 

```js
 yield take(
      FETCH_ACTIVE_PARCEL_MORTGAGES_REQUEST,
      handleFetchActiveParcelMortgagesRequest
    )
```

Reply:
done